ui/src/app/components/cases/list.component.ts

```diff
@@ -75,7 +76,7 @@ export class TestCasesListComponent extends BaseComponent implements OnInit {
   }
 
   get isStepGroup() {
-    return this.router.url.indexOf("/step_groups") > -1;
+    return this.router.url.indexOf("/step_groups") !== -1;
```

Comment:
what is the need for change in logical expression

Reply:
-1 is the value for specifying that its not a step group. step group index can be lesser than or greater than -1. The same logical expression is also used in cloud. This is an enhancement from cloud. @guganabu

Reply:
ideally, -1 for not existence and anything more than that will refer to existence of lookup value. so, > -1 should work as !== -1